content/04.use-cases-b.md

```diff
@@ -1,99 +1,90 @@
+RMarkdown via the `bookdown` R package (<https://bookdown.org/>), Jupyter notebooks [@doi:10.1038/s42005-020-00403-4], Quarto (<https://quarto.org>)], scientists can adopt an unified dynamic and collaborative framework that allows prose and data analysis to be combined in centralized, versioned repository.
+
+### Peer review
 
 <!--*contributors to this section:* Eric R. Scott-->
-Peer review plays a critical role in the scientific evaluation of EEB research.
-GitHub provides an open and transparent platform that can be used for either directly providing feedback on research products or building a to-do list from reviewer comments.
-Peer review of research code by the open software development community rOpenSci (<https://ropensci.org/software-review/>) and of research software and associated manuscripts by the Journal of Open Source Software (<https://joss.readthedocs.io/en/latest/submitting.html>) requires that submitted work is hosted on GitHub and their review processes make use of GitHub issues.
-rOpenSci's efforts have resulted in many well-used R packages for ecology research including rfishbase [@doi:10.1111/j.1095-8649.2012.03464.x] and taxize [@doi:10.12688/F1000RESEARCH.2-191.V2].  
-
-GitHub can also be used as a hub for reviewers and authors during the peer review process of an ordinary research manuscript.
-If the code associated with a manuscript is made available at the time of submission (e.g. via a link to a GitHub repository in a Data Availability Statement), peer-reviewers may be able to offer more helpful suggestions on written methods and may even make comments on the code itself, potentially catching bugs or errors before publication.
-GitHub can also be used as a hub for reviewers and authors during the peer review process of an ordinary research manuscript.
-GitHub issues can be used to organize and discuss reviewer suggestions and to assign them to co-authors (See example in <https://github.com/BrunaLab/HeliconiaDemography/issues?q=is%3Aissue+label%3A%22reviewer+comment%22+>).
+Peer review is the standard process for assessing and judging whether research done in ecology and evolution should be published in a scientific journal.
+GitHub provides an open and transparent platform that can be used for either directly providing feedback on research products or addressing changes recommended by reviewers.
+GitHub Issues can be used to organize and discuss reviewer suggestions and to assign them to co-authors (_e.g._, <https://github.com/BrunaLab/HeliconiaDemography/issues?q=is%3Aissue+label%3A%22reviewer+comment%22+>).
 When reviewer comments are posted as separate issues, authors can comment on the issues to discuss possible changes and assign themselves to indicate which comments they intend to handle.
 Co-authors can then integrate their edits and responses to reviewers using pull requests.
+GitHub can also serve to reviewers and authors during the peer review process.
```

Comment:
"can aslo assist reviewers" (?) I am not sure if that is what is meant - maybe "serve reviewers" could work but not with the addition of "to"